client/signup/steps/site-or-domain/index.jsx

```diff
@@ -54,11 +54,16 @@ class SiteOrDomain extends Component {
 
 		const choices = [];
 
+		const buyADomainDescription =
+			i18n.getLocaleSlug() === 'en' || i18n.hasTranslation( 'Add a site later.' )
```

Comment:
Nowadays we also have [the useHasEnTranslation()](https://github.com/Automattic/wp-calypso/pull/80106/files#diff-20bd370d798933f546a5225a3aa423558cda498f19c4f7199e165cd818cf1cfeR20) as an option, but no need to change it here. It's probably more useful when using inside the component rendering and has one use in the file only.

Reply:
Thanks, Paulo! I believe I tried [useHasEnTranslation()](https://github.com/Automattic/wp-calypso/pull/80106/files#diff-20bd370d798933f546a5225a3aa423558cda498f19c4f7199e165cd818cf1cfeR20) but my IDE threw the error:

> ESLint: React Hook "useHasEnTranslation" cannot be called in a class component. React Hooks must be called in a React function component or a custom React Hook function.(react-hooks/rules-of-hooks)

So I ended up with the current implementation. 😅